### PR TITLE
PROD-137: Make changes to our application

### PR DESCRIPTION
**Pull request by NickAnderegg**

### CODE REVIEW

File: preorderTraversal.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  this.left = this.right = null;
+}
+
+function preorderTraversal(root) {
+  let result = [];
+  if (root !== null) {
+    result.push(root.val);
+    preorderTraversal(root.left);
+    preorderTraversal(root.right);
+  }
```
[the remark this answers is not in the document]
Comment (NickAnderegg) [reply preorderTraversal.js:12]: Could you provide me with an example of what the suggested change will look like in context?

[the remark this answers is not in the document]
Comment (NickAnderegg) [reply preorderTraversal.js:12]: Could you provide a version of the suggested change that doesn't introduce a new function?

[the remark this answers is not in the document]
Comment (NickAnderegg) [reply preorderTraversal.js:12]: What is the simplest way we could improve the code to be in line with your suggestions?

**Comment by kc-dot-io** [comment]
@NickAnderegg Can this be closed now?

**Pull request closed without merging**